packages/terra-collapsible-menu-view/tests/jest/__snapshots__/CollapsibleMenuViewItem.test.jsx.snap

```diff
@@ -29,19 +29,13 @@ exports[`CollapsibleMenuViewItem Collapsible Menu Context should not set icon on
       "timeZone": null,
     }
   }
-  isActive={false}
-  isDisabled={false}
-  isHighlighted={false}
-  isInstructionsForUse={false}
```

Comment:
Are these attributes removed from menu-item ?? if not can you use .dive() for snapshot and see if you can spot these attributes

Reply:
Used dive, attributes are back: https://github.com/cerner/terra-framework/pull/1626/commits/223edd4ef08fea43e4f8810c1a5872170b1a1cd8